cumulus_library/template_sql/column_datatype.sql.jinja

```diff
@@ -1,8 +1,13 @@
 SELECT
+    {%- if include_table_names %}
+    table_name,
+    {%- endif %}
     column_name,
     data_type
 FROM information_schema.columns
 WHERE
     table_schema = '{{ schema_name }}'
-    AND table_name = '{{ table_name|lower }}'
-    AND LOWER(column_name) IN ('{{ column_names|join("', '")|lower }}') --noqa: LT05
+    AND table_name IN ('{{ table_names|join("', '")|lower }}')
+{%- if column_names %}
+AND LOWER(column_name) IN ('{{ column_names|join("', '")|lower }}') --noqa: LT05
```

Comment:
Why isn't this indented? I don't care, but I thought this repo liked its sensible SQL output.

Reply:
I truly cannot figure out why sqlfluff was complaining about this, and end of day brain - the other option is another noqa statement

Reply:
noqa it is for this